app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/non_calculated_enter_tax_due.scala.html

```diff
@@ -46,24 +53,38 @@
   else if(wasUkResident) ".ukResident" -> viewConfig.calculateCgt
   else ".nonResident" -> viewConfig.calculateCgtNr
 }
+@label = @{messages(s"$key.label")}
```

Comment:
⛏️ I'd prefer not assembling messages' keys from separate strings but rather use whole strings so they're easier to locate in messages.

Reply:
I wanted to follow the existing pattern, but yes, I agree, using compound key expressions makes it more difficult to find references